### PR TITLE
Show both open & cancelled order transaction hashes for cancelled open orders

### PR DESCRIPTION
**Pull request by adrake33**
[Clubhouse Story](https://app.clubhouse.io/augur/story/11557/when-cancelling-an-open-order-the-transaction-page-only-displays-a-tx-hash-for-original-order-not-for-cancelled-transaction)

Please refer to the screenshot Matt posted in the comments section of the CH issue to see how cancelled open orders should now be displayed on the Portfolio: Transactions page.

## Submitter checklist
- [ ] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [ ] Steps to reproduce
  - [x] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass

### CODE REVIEW

File: src/modules/portfolio/components/transaction-linked/transaction-linked.styles.less [hunk 1]
```diff
@@ -36,6 +36,13 @@
   font-weight: 700;
 }
 
+.LinkedTransaction__message-cancelled {
+  color: #ff0068;
```
Comment (phoebemirman) [inline src/modules/portfolio/components/transaction-linked/transaction-linked.styles.less]: can you use the color constant here?

Comment (adrake33) [reply src/modules/portfolio/components/transaction-linked/transaction-linked.styles.less]: Whoops, yeah I'll change that.

**Comment by sharkcrayon** [comment]
See last minute request in CH task.

**Comment by adrake33** [comment]
@sharkcrayon Sounds good, will add that to this PR.

**Pull request merged**